### PR TITLE
update news

### PR DESCRIPTION
**Pull request by benoit-pierre**
If think everything is ready for a new release, so here is an tentative version of the release notes for the next version.

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -2,6 +2,36 @@
 
                       NEWS for Python X Library
 
+* Version 0.16
+
+** Licensing
+
+The project is now licensed under the GNU Lesser General Public License
+v2 or later (LGPLv2+).
```
Comment (benoit-pierre) [inline NEWS]: OK, amended.

**Comment by benoit-pierre** [comment]
OK, new release?

**Comment by vasily-v-ryabov** [comment]
Agree. Everything seems ready.

**Comment by benoit-pierre** [comment]
Great, who's doing it? (the Github part)

**Comment by vasily-v-ryabov** [comment]
Please proceed with all parts.

**Comment by benoit-pierre** [comment]
OK, all done. I had 2 (minor) issues:
* the GitHub token was wrong (I think Travis had auto-detected the wrong repository, using mine instead of `python-xlib/python-xlib`)
* I had to generate and upload another source distribution format because `tar.xz` is not supported by PyPi; I'll check what formats are supported and make a PR to update our default format so the next release goes smoothly

**Comment by vasily-v-ryabov** [comment]
Cool! Thanks a lot for your outstanding efforts!

As I remember, PyPI supports `.tar.gz` and some `.zip` formats (built with `sdist --fotmat=zip` option for `setup.py`). Probably it checks only 1 compression level for every compression format. For example, manually zipped files with all perks enabled are not accepted by PyPI for Windows packages. I think similar things take place for Linux.

**Pull request merged**